### PR TITLE
📈 Implement Missed/Missing Predictions Dashboard

### PR DESCRIPTION
**Pull request by Whoops**
#### Summary of changes
**Asana Ticket:** [📈 Implement Missed/Missing Predictions Dashboard](https://app.asana.com/0/584764604969369/1206011951284569/f)

#### Reviewer Checklist
- [ ] Meets ticket's acceptance criteria
- [ ] Any new or changed functions have typespecs
- [ ] Tests were added for any new functionality (don't just rely on Codecov)

### CODE REVIEW

File: lib/prediction_analyzer_web/templates/terminal_departure/index.html.eex [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            <td><%= format_unix(departure_time) %></td>
+            <td><%= format_unix(min_file_time) %></td>
+            <td><%= format_unix(max_file_time) %></td>
+          </tr>
+          <% end %>
+        </tbody>
+      </table>
+    <% true -> %>
+    <div class="col-sm-6">
+    <h3>Upredicted Departures</h3>
```
Comment (meagharty) [inline lib/prediction_analyzer_web/templates/terminal_departure/index.html.eex:71]: typo: Unpredicted 👍 
```suggestion
    <h3>Unpredicted Departures</h3>
```

File: lib/prediction_analyzer/filters/stop_groups.ex [hunk 2]
```diff
@@ -9,7 +9,7 @@ defmodule PredictionAnalyzer.Filters.StopGroups do
   @c_branch ~w(70211 70212 70213 70214 70215 70216 70217 70218 70219 70220 70223 70224 70225 70226 70227 70228 70229 70230 70231 70232 70233 70234 70235 70236 70237 70238)
   @d_branch ~w(70160 70161 70162 70163 70164 70165 70166 70167 70168 70169 70170 70171 70172 70173 70174 70175 70176 70177 70178 70179 70180 70181 70182 70183 70186 70187)
   @e_branch ~w(70239 70240 70241 70242 70243 70244 70245 70246 70247 70248 70249 70250 70251 70252 70253 70254 70255 70256 70257 70258 70260)
-  @terminals ~w(70001 70036 70038 70059 70060 70061 70093 70094 70105 70106 70107 70160 70161 70196 70197 70198 70199 70201 70202 70205 70206 70237 70238 70260 70261 70275 70276 70838 71199)
+  @terminals ~w(70001 70036 70038 70059 70061 70094 70105 70106 70160 70161 70196 70197 70198 70199 70201 70202 70205 70206 70238 70260 70261 70276 71199 70503)
```
Comment (Whoops) [inline lib/prediction_analyzer/filters/stop_groups.ex:12]: Per @fsaid90 these Exit Only stops should not be considered terminals for the new or existing dashboards.

**Comment by Whoops** [comment]
Currently deployed to https://prediction-analyzer-dev.mbtace.com/missed_predictions

File: test/prediction_analyzer_web/controllers/missed_predictions_controller_test.exs [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+      assert response =~ "Unpredicted Departures"
+      assert response =~ "Unrealized Departures"
+    end
+
+    test "Generates totals for missed and unpredicted departures", %{conn: conn} do
+      conn = get(conn, "/missed_predictions", date: "2020-02-01", env: "prod")
+      response = html_response(conn, 200)
+
+      assert response =~ "<strong>Total</strong>"
```
Comment (Whoops) [inline test/prediction_analyzer_web/controllers/missed_predictions_controller_test.exs:112]: This is kind of a funky way of checking that the totals came out correctly, but for this simple function I think it's sufficient.

**Pull request merged**